docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
 
 This example used a PI Server as the destination, however the same mechanism and filters may be used for any north destination.
 
+Graphical Pipeline Development
+------------------------------
+
+In versions 1.0 up until version 2.4 of Fledge the user interface for creating and editing pipeline was a purely grid based interface as illustrated above. In version 2.4 an option was introduced that allowed a more graphical approach to creating and managing data pipelines. This option was activated via the *Settings* menu option and is call the *Flow editor*. Enabling this will change the user interface for pipeline management.
+
+.. note::
+
+   As of version 3.0 the graphical, flow editor will be the default user interface mode. The previous interface is still available and may be selected by turning off the flow editor in the Settings menu.
+
+Adding A Pipeline
+~~~~~~~~~~~~~~~~~
+
+Adding a pipeline, in for example the south, is done by navigating to the *South* menu item, a page will appear that shows all of the current south service, or pipeline, that have been configured.
+
++--------------+
+| |flow_south| |
++--------------+
+
+.. note::
+
+   The north services and tasks are presented in the same way and the interactions are the same as is described here for the south services.
+
+To add a new service, click on the + icon in the dotted service in the top left corner. You will then be presented with a dialog that allows you to choose the south plugin to use to ingest data. If working on a north service or task you Will choose the plugin to use to send data to the system north of Fledge.
```

Comment:
- What is the "dotted service?" Do you mean dotted menu?
- task you Will choose -> task you will choose